priv/repo/migrations/20210421205214_create_organizers.exs

```diff
@@ -5,6 +5,8 @@ defmodule Bokken.Repo.Migrations.CreateOrganizers do
     create table(:organizers, primary_key: false) do
       add :id, :binary_id, primary_key: true
       add :champion, :boolean, default: false, null: false
+      add :first_name, :string, null: false
+      add :last_name, :string, null: false
```

Comment:
You need to create an ALTER migration. Don't forget that bokken is already in prod 😅